website/docs/r/stream_analytics_output_powerbi.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+subcategory: "Stream Analytics"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_stream_analytics_output_powerbi"
+description: |-
+  Manages a Stream Analytics Output powerBI.
+---
+
+# azurerm_stream_analytics_output_powerbi
+
+Manages a Stream Analytics Output powerBI.
+
+## Example Usage
+
+```hcl
+data "azurerm_resource_group" "example" {
+  name = "example-resources"
+}
+
+data "azurerm_stream_analytics_job" "example" {
+  name                = "example-job"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_stream_analytics_output_powerbi" "example" {
+  name                      = "output-to-powerbi"
+  stream_analytics_job_name = azurerm_stream_analytics_job.example.name
+  resource_group_name       = azurerm_stream_analytics_job.example.resource_group_name
+  dataset                   = "example-dataset"
+  table                     = "example-table"
+  group_id                  = "00000000-0000-0000-0000-000000000000"
+  group_name                = "some-group-name"
+  authentication_mode       = "Msi"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Stream Output. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Stream Analytics Job exists. Changing this forces a new resource to be created.
+
+* `stream_analytics_job_name` - (Required) The name of the Stream Analytics Job. Changing this forces a new resource to be created.
```

Comment:
this needs to be updated to reflect schema?

Reply:
Thanks for reminding. Will update.